### DISCUSSION TITLE
Just so I understand

### DISCUSSION THREAD

**Discussion by eric859** [itsToggle/plex_debrid#377] [Q&A]
Example: 100 movies in trakt collection and plex_debrid is setup correctly:

When I start both rclone and plex_debrid, it looks at the trakt collection and if there is nothing downloaded in x:\movies for each movie, it will download it to that location 1 at a time until all 100 are done.  Once it's downloaded and Plex sees the media file, Plex will add it to the Plex library.  The movie is streamed from the file downloaded in your real-debrid x: drive.

The way it works in Kodi is that your addon adds movies to the library by creating an srt file for each movie.  When played, the kodi addon is called to search for torrents and you select from any one of the direct links to start streaming.  I don't know what happens in real-debrid in the background, but I don't think it downloads a media file to your RD movies folder... it just streams from the link. 

The main difference is that plex_debrid will take a long time downloading each movie to your RD movie folder before it's even added to plex. 

Am I correct?

**Comment by rebsii** [comment]
It works similar to Kodi, in that rclone is holding the "links" virtually in a drive and plex will pick them up for streaming to the client. Nothing is downloaded to a folder.

**Comment by itsToggle** [comment]
It will add whatever you tell it to add. If you dont force it to add large amounts of movies before hand, but just add movies as you want to watch them, it takes ~20 seconds between watchlisting something and you being able to watch it via Plex.
